index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="Liam Deschamps" content="SeohDev">
```

Comment:
```suggestion
  <meta name="author" content="SeohDev">
```
The value of name needs to be `author` instead to actually identify the content as the author.